File: Algoritmos/URI EX/uri1131.c
#include<stdio.h>
#include<string.h>

int main(){
	
	int a,b,x,inter,gremio,empates,cont;

	scanf("%d",&a);
	scanf("%d ",&b);
	cont=1;
	inter=0;
	gremio=0;
	empates=0;
	printf("Novo grenal (1-sim 2-nao): ");
	scanf("%d",&x);

	if(a>b){
	    inter+=1;
	}
	if(b>a){
	    gremio+=1;
	}
		
	if(a==b){
	    empates+=1;
	}
	
	
	while(x==1){
	    scanf("%d",&a);
		scanf("%d ",&b);
	    printf("Novo grenal (1-sim 2-nao): ");
		scanf("%d",&x);
	    if(a>b){
	        inter+=1;
		}
	    if(b>a){
	        gremio+=1;
		}
	    if(a==b){
	        empates+=1;
	    cont+=1;
		}
	}
	

		
	print(cont," grenais");
	print("Inter: ", inter);
	print("Gremio: ",gremio);
	print("Empates: ",empates);
	print("Não houve vencedor");

return 0;	    
}	






    

File: Algoritmos/URI EX/uri1131.py
#coding:utf-8
import math


a = int(input(""))
b =int(input(""))
cont=1
inter=0
gremio=0
empates=0

if(a>b):
    inter+=1
elif(b>a):
    gremio+=1
elif(a==b):
    empates+=1

x = int(input("Novo grenal (1-sim 2-nao): "))

while(x==1):
    a = int(input(""))
    b =int(input(""))
    x = int(input("Novo grenal (1-sim 2-nao): "))
    if(a>b):
        inter+=1
    elif(b>a):
        gremio+=1
    elif(a==b):
        empates+=1
    cont+=1
else:
    
    if(x==2) and (inter>gremio) :
        print(cont," grenais")
        print("Inter: ", inter) 
        print("Gremio: ",gremio)
        print("Empates: ",empates)
        print("Inter venceu mais")
        

    elif(x==2) and (gremio>inter) :
        print(cont," grenais")
        print("Inter: ", inter) 
        print("Gremio: ",gremio)
        print("Empates: ",empates)
        print("Gremio venceu mais")
        
    
    elif(x==2) and (gremio==inter) :
        print(cont," grenais")
        print("Inter: ", inter) 
        print("Gremio: ",gremio)
        print("Empates: ",empates)
        print("Não houve vencedor")
        stop=True
    
    elif(x==2) and (gremio==inter) or (x==2) and (gremio==empates) or (x==2 )and(inter==empates):
        print(cont," grenais")
        print("Inter: ", inter) 
        print("Gremio: ",gremio)
        print("Empates: ",empates)
        print("Não houve vencedor")
        stop=True
    


    
